main.py

```diff
@@ -504,7 +506,7 @@ def main() -> None:
     parser.add_argument(
         "-j",
         "--libjs-test262-runner",
-        required=True,
+        default="./Build/libjs-test262-runner",
```

Comment:
Also added this, which is a super old commit I had lying around. Let me know if you disagree with this change I can just remove it again.

Reply:
Nope, that's great :+1: